addons/web/static/src/js/framework/session.js

```diff
@@ -141,9 +141,12 @@ var Session = core.Class.extend(mixins.EventDispatcherMixin, {
         }
         var def = this._groups_def[group];
         if (!def) {
-            var Model = window.openerp.web.Model;
-            var Users = new Model('res.users');
-            def = this._groups_def[group] = Users.call('has_group', [group]);
+            def = this._groups_def[group] = this.rpc('/web/dataset/call_kw/res.users/has_group', {
```

Comment:
hmm, how does this work? you explicitely use 'base.group_no_one' and not the variable group?